gpiozero/boards.py

```diff
@@ -1186,7 +1186,45 @@ def off(self):
         self.value = False
 
 class SevenSegmentDisplay(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for a 7 segment LED display 
+
+    7 segment displays have either 7 or 8 pins, 7 pins for the digit display 
+    and an optional 8th pin for a decimal point. 7 segment displays 
+    typically have either a common anode or common cathode pin, when
+    using a common anode display 'active_high' should be set to False.
+    Instances of this class can be used to display characters or control 
+    individual leds on the display. For example::
+
+        from gpiozero import SevenSegmentDisplay
+
+        seven = SevenSegmentDisplay(1,2,3,4,5,6,7,8,active_high=False)
+        seven.display("7")
```

Comment:
'seven display seven' sounds a bit tautological ;-)
Perhaps `seven.display("5")`

And GPIO1 is one of the 'reserved' pins, so maybe pins 2-9 would be better? (only a minor point)